src/bits/bit_array.rs

```diff
@@ -252,3 +252,85 @@ macro_rules! bit_array_impl {
 
 bit_array_impl!(bit_array_64, BitArray64, U8_8, 64);
 bit_array_impl!(bit_array_40, BitArray40, U8_5, 40);
+
+impl Serializable for BitArray40 {
```

Comment:
is there a reason we couldn't implement this inside `bit_array_impl`?

Reply:
no real reason, I thought it won't be possible because the size of the bit array *store* needs to be known at compile time, but Rust seems to be handling it just fine